src/user_management/user.resolver.js

```diff
@@ -2,8 +2,22 @@ import models from '~/src/service_providers/sequelize/models'
 
 export default {
   Mutation: {
-    createUser: () => {
-      return models.user.create()
+    createUser: (_,{input}) => {
+      return models.user.create({
+        firstName: input.firstName,
+        lastName: input.lastName,
+        email: input.email,
+        password: input.password
+      })
+    }
+  },
+  Query: {
+    logInUser: (email, password) => {
+      var pass = models.user.generateHash(password);
+      return models.user.find(user => user.email === email && user.password === pass);
+    },
```

Comment:
- This won't work. You need to find the user first and then verify the password with the method you created.
- Sequelize models don't have a `find` method. Please read the documentation for [`findOne`](https://sequelize.org/master/class/lib/model.js~Model.html#static-method-findOne) for more information on how to write a query with Sequelize.
- Don't use `var`! Are you running the linter ([StandardJS](https://standardjs.com/), from block 2) before uploading your code?
- Don't use semicolons (`;`)!
```suggestion
    logInUser: async (email, password) => {
      const user = await models.user.findOne({ where: { email } })
      
      if (!await user?.validatePassword(password)) {
        throw new Error('Invalid email or password')
      }
      
      return user
    },
```